crates/proto/src/serialize/mod.rs

```diff
@@ -17,3 +17,5 @@
 //! Contains serialization libraries for `binary` and text, `txt`.
 
 pub mod binary;
+#[cfg(feature = "text-parsing")]
```

Comment:
can we add this here for docs.rs:

`#[cfg_attr(docsrs, doc(cfg(feature = "text-parsing")))]`

Reply:
@djc, I'll take care of this in a different commit. I can merge this as is...

Reply:
oh, there's a conflict... I'll take care of this.